app/views/clients/_default_tab_navigation.haml

```diff
@@ -104,6 +104,18 @@
         permission: can_use_service_register?,
       },
     }) if RailsDrivers.loaded.include?(:service_scanning) && GrdaWarehouse::Config.get(:service_register_visible)
+
+    supplemental_data_sets = HmisSupplemental::DataSet.viewable_by(current_user).where(data_source_id: @client.source_clients.select(:data_source_id))
+    supplemental_data_sets.each do |data_set|
+      key = hmis_supplemental_data_set_client_data_set_path(id: @client.id, data_set_id: data_set.id)
+      hmis.merge!({
+        key => {
+          title: data_set.name,
+          permission: :can_view_supplemental_client_data?,
+        }
+      })
+    end
```

Comment:
Since this is coming from a driver, to be good citizens, we need to wrap calls in the main app with something like `if RailsDrivers.loaded.include?(:hmis_supplemental)`.

I didn't call it out in the other files, but if the code is referencing a driver, we should be making sure it _can_ be run without the driver.